rest-api-spec/src/main/resources/rest-api-spec/test/search.aggregation/40_range.yml

```diff
@@ -528,3 +528,31 @@ setup:
   - is_false:  aggregations.unsigned_long_range.buckets.2.to
 
   - match: { aggregations.unsigned_long_range.buckets.2.doc_count: 0 }
+
+---
+"Empty Agg Body":
```

Comment:
Thanks for adding a failing test!

Nitpick, but why include this case in the `range` test?

Reply:
Also, can we add some sort of match to the test, like maybe on `hits.total`?

Reply:
Actually this file is for `aggregation` test with ranges. Full path includes the `search.aggregation` folder: `rest-api-spec/src/main/resources/rest-api-spec/test/search.aggregation/40_range.yml`

Reply:
@jed326 we could move it off to separate file like `60_empty.yml`